fiftyone/operators/delegated.py

```diff
@@ -90,6 +90,8 @@ def set_running(self, doc_id, progress=None, run_link=None):
                 operation
             run_link (None): an optional link to orchestrator-specific
                 information about the operation
+            required_state (ExecutionRunState): an optional required state of the operation. If provided, the
+                update will only be performed if the referenced operation matches this state.
 
         Returns:
             a :class:`fiftyone.factory.repos.DelegatedOperationDocument`
```

Comment:
Recommend updating return section on all set_* methods docstrings to say something like could be `None` if no such run is found, particularly in the case that `required_state` is set.